controllers/storagecluster/initialization_reconciler_test.go

```diff
@@ -393,13 +391,18 @@ func createFakeInitializationStorageClusterReconciler(t *testing.T, obj ...runti
 		}
 	}
 
-	runtimeObjects = append(runtimeObjects, mockNodeList.DeepCopy(), cbp, cfs, cnfs, cnfsbp, cnfssvc, infrastructure, networkConfig, rookCephMonSecret, csv)
+	runtimeObjects = append(runtimeObjects, mockNodeList.DeepCopy(), cbp, cfs, cnfs, cnfsbp, cnfssvc, infrastructure, networkConfig, rookCephMonSecret, csv, createVirtualMachineCRD())
```

Comment:
This is a very long line. can you please split into a line per item?
```suggestion
	runtimeObjects = append(
		runtimeObjects, 
		mockNodeList.DeepCopy(), 
		cbp, 
		cfs, 
		cnfs, 
		cnfsbp, 
		cnfssvc, 
		infrastructure, 
		networkConfig, 
		rookCephMonSecret, 
		csv, 
		createVirtualMachineCRD(),
	)
```